docs/source/user-guide/user_interface/integrations.rst

```diff
@@ -0,0 +1,122 @@
+=============
+Integrations
+=============
+
+
+Notifications
+=============
+
+
+Email Notifications
+-------------------
+
+To configure email notifications go to the page 'Configurations/Integrations' and click on the tab 'Notifications'.
+On the shown table mark alert severity levels about which you want to be notified.
```

Comment:
@yromanyshyn Replace with  "On the display table mark the alert severity levels for which you would like to receive notifications on"

Reply:
fixed